src/Ledger/Ledger.lagda

```diff
@@ -79,7 +79,7 @@ data
   LEDGER : let open LState s; txb = tx .body; open TxBody txb; open LEnv Γ in
     ∙  record { LEnv Γ } ⊢ utxoSt ⇀⦇ tx ,UTXOW⦈ utxoSt'
     ∙  ⟦ epoch slot , pparams , txvote , txwdrls ⟧ᶜ ⊢ certState ⇀⦇ txcerts ,CERTS⦈ certState'
-    ∙  ⟦ txid , epoch slot , pparams , enactState ⟧ᵍ ⊢ govSt ⇀⦇ txgov txb ,GOV⦈ govSt'
+    ∙  ⟦ txid , epoch slot , pparams , ppolicy , enactState ⟧ᵍ ⊢ govSt ⇀⦇ txgov txb ,GOV⦈ govSt'
```

Comment:
Where does this policy come from? Shouldn't it be the one in the constitution?

Reply:
`LEnv` already has this field, and this is exactly the one in the constitution.